src/components/views/elements/DesktopBuildsNotice.tsx

```diff
@@ -37,19 +37,22 @@ export default function DesktopBuildsNotice({ isRoomEncrypted, kind }: IProps) {
     if (EventIndexPeg.get()) return null;
 
     if (EventIndexPeg.error) {
-        return <>
-            { _t("Message search initialisation failed, check <a>your settings</a> for more information", {}, {
-                a: sub => (<a onClick={(evt) => {
-                    evt.preventDefault();
-                    dis.dispatch({
-                        action: Action.ViewUserSettings,
-                        initialTabId: UserTab.Security,
-                    });
-                }}>
-                    { sub }
-                </a>),
-            }) }
-        </>;
+        return (
+            <div className="mx_DesktopBuildsNotice">
```

Comment:
The class name is now slightly misleading.
Nothing here really is linked to desktop builds.

It would be good to rename this class and also probably move everything away from `_DesktopBuildsNotice.scss` to either the search or room view scss files as this is now becoming a generic warning class

Reply:
I renamed the class and scss files to be `SearchWarning`. Is this sufficient or should we also merge the scss file into one of the other two?